django-tethne/models.py

```diff
@@ -0,0 +1,163 @@
+from django.db import models
+
+
+
+"""
+Model Name = Source
+
+Source represents the source of the Corpus
+Example 1. JSTOR, 2. ZOTERO or 3. WOS
+"""
+class Source(models.Model):
+    source_value = models.CharField(max_length=35, null=True)
```

Comment:
You can constrain the values here by using the ``choices`` parameter. See: https://docs.djangoproject.com/en/1.9/ref/models/fields/#choices

Also, let's using something more semantically meaningful here. E.g. ``name`` would make sense, since you'll refer to the field using ``sourceInstance.name``